src/number_theory/legendre_symbol/quadratic_reciprocity.lean

```diff
@@ -1,16 +1,13 @@
 /-
-Copyright (c) 2018 Chris Hughes. All rights reserved.
+Copyright (c) 2018 Chris Hughes, 2022 Michaael Stoll. All rights reserved.
```

Comment:
Afaict, the first line of the copyright header doesn't change after a file is created. But it's perfectly fine (and encouraged) to add yourself to the list of authors if you make substantial changes/additions to a file.
```suggestion
Copyright (c) 2018 Chris Hughes. All rights reserved.
```

Reply:
OK, I didn't know that. Changed back.